src/Umbraco.Core/Models/IFile.cs

```diff
@@ -40,6 +40,14 @@ public interface IFile : IAggregateRoot
         string Content { get; set; }
 
         /// <summary>
+        /// Indicates that the file should use whatever content already
+        /// exists on the filesystem which manages the file, bypassing content
+        /// management entirely.
+        /// </summary>
+        /// <remarks>Only for the next save. Is resetted when the file is saved.</remarks>
+        void UseExistingContent();

```

Comment:
I understand that we don't want to break the interfaces. And then. I need a way to create an IFile (database object), while _reusing_ whatever actual file exists on disk. I'll see how I can do it without altering IFile. Maybe Deploy can use a special class/interface that inherits/implements IFile and adds more stuff.